graspy/embed/omni.py

```diff
@@ -157,7 +157,9 @@ def fit(self, graphs, y=None):
         self : object
             Returns an instance of self.
         """
-        graphs = self._check_input_graphs(graphs)
+        graphs = import_multigraphs(graphs)
```

Comment:
Are we sure we want to use the name multigraphs for a series of graphs?  I ask because networkx has multigraphs as an actual thing and using the same nomenclature implies certain things https://networkx.github.io/documentation/networkx-1.9.1/reference/classes.multigraph.html

Reply:
Yea these are multigraphs. Multigraph just means each pair of nodes can have more than 1 edge. Any of the algorithms that work on multiple graphs is a multigraph algorithm. In this particular case, the edges have labels and are ordered.

In algos like omni and mase, the edges are just labeled, but not ordered. This means that you can shuffle the orderings and nothing really changes.